Provider/ImageProvider.php

```diff
@@ -174,6 +174,10 @@ public function generatePublicUrl(MediaInterface $media, $format)
      */
     public function generatePrivateUrl(MediaInterface $media, $format)
     {
+        if ($format == 'reference') {
```

Comment:
I'm fine with making those changes. However, I do question why a Yoda condition is being asked for with this when the rest of the bundle doesn't seem to use them. It was written this way to maintain consistency with the rest of the bundle.

For example: https://github.com/sonata-project/SonataMediaBundle/blob/master/Provider/FileProvider.php#L364-369